src/EditorFeatures/Core.Wpf/InlineDiagnostics/InlineDiagnosticsTaggerProvider.cs

```diff
@@ -34,7 +35,7 @@ internal class InlineDiagnosticsTaggerProvider : AbstractDiagnosticsAdornmentTag
         private readonly IClassificationFormatMapService _classificationFormatMapService;
         private readonly IClassificationTypeRegistryService _classificationTypeRegistryService;
 
-        protected sealed override IEnumerable<PerLanguageOption2<bool>> PerLanguageOptions => SpecializedCollections.SingletonEnumerable(InlineDiagnosticsOptions.EnableInlineDiagnostics);
+        protected sealed override ImmutableArray<IOption> FeatureOptions { get; } = ImmutableArray.Create<IOption>(InlineDiagnosticsOptions.EnableInlineDiagnostics);
```

Comment:
I think I understand the intent behind `FeatureOptions` vs `Options`, and I think you're preserving the existing behavior here. But I don't think `EnableInlineDiagnostics` changes how the feature behaves. It's just turning the feature on and off.

To me, it looks like this should be `Options` not `FeatureOptions`, and in that case, there will remain zero overrides of `FeatureOptions` and it can be removed.

Reply:
Oh I missed this, I agree with that `EnableInlineDiagnostics` should be Options, not FeatureOptions. The feature specific option is the `InlineDiagnosticsOptions.Location`

Reply:
sorry, you caught me mid-edit and i was pulled into another convo.  THis has been resolved.